lib/models/BucketLoggingStatus.ts

```diff
@@ -0,0 +1,193 @@
+import { parseString } from 'xml2js';
+import errors, { ArsenalError, errorInstances } from '../errors';
+
+/** BucketLoggingStatus constants, not documented by AWS but found via testing */
+const bucket_logging_status_target_bucket_min_length = 3;
+const bucket_logging_status_target_bucket_max_length = 255;
+const bucket_logging_status_target_prefix_max_length = 800;
```

Comment:
because they are specific to this field you don't need to prefix with bucket logging.
And I'd say uppercase is prefered for constants

```suggestion
const TARGET_BUCKET_MIN_LENGTH = 3;
const TARGET_BUCKET_MAX_LENGTH = 255;
const TARGET_PREFIX_MAX_LENGTH = 800;
```

Reply:
done